fabric-rendering-fluids-v1/src/main/resources/fabric.mod.json

```
@@ -19,7 +19,8 @@
     "fabric-api-base": "*"
   },
   "recommends": {
-    "fabric-textures-v0": "*"
+    "fabric-textures-v0": "*",
+    "fabric-fluid-v1": "*"
```

```suggestion
    "fabric-fluid-api-v1": "*"
```
also IMO that comment about `suggests` applies here as well (especially since you can make a fully functional custom fluid without that module)